test/lib/base-controller.js

```diff
@@ -329,7 +329,67 @@ describe('lib/base-controller', function () {
             }];
             controller.options.continueOnEdit = false;
             req.params.action = 'edit';
-            controller.getNextStep(req).should.contain('/confirm');
+            controller.getNextStep(req).should.equal('/confirm');
+          });
+
+          it('should return /a-base-url/confirm if baseUrl is set', function () {
+            getStub.returns(['/target-page']);
+            req.form.values['example-radio'] = 'superman';
+            controller.options.forks = [{
+              target: '/target-page',
+              condition: function (request) {
+                return request.form.values['example-radio'] === 'superman';
+              }
+            }];
+            controller.options.continueOnEdit = false;
+            req.params.action = 'edit';
+            req.baseUrl = '/a-base-url';
+            controller.getNextStep(req).should.equal('/a-base-url/confirm');
+          });
+
+          it('should return /a-base-url/target-page if completed check falsy', function () {
+            getStub.returns(['/a-base-url/target-page']);
```

Comment:
Wouldn't it make more sense for this line to be?:
getStub.returns([]);

(As your point in the PR is that the list of steps do NOT include the baseUrl.)

In fact, does this test (and the one below it) test anything new. Perhaps they should be removed?

@JoeChapman: Thoughts?